KsApi/models/Project.swift

```diff
@@ -71,6 +72,14 @@ public struct Project {
     }
   }
 
+  public struct LiveStream {
+    public let id: String
```

Comment:
are we certain this always comes back as a string? if it sometimes comes as an int we need to do a little work to decode as both